R/utilities-export.R

```diff
@@ -0,0 +1,13 @@
+#' @export
+exportResultsToCSV <- function(results, simulation, filename){

```

Comment:
trivial method to export results and pk analysis to csv
Example of output

IndividualId | Quantity Path | Parameter | Value | Unit
-- | -- | -- | -- | --
0 | Organism\|PeripheralVenousBlood\|Caffeine\|Plasma (Peripheral Venous Blood) | C_max | 20.15721321 | µmol/l
0 | Organism\|PeripheralVenousBlood\|Caffeine\|Plasma (Peripheral Venous Blood) | C_max_norm | 3914530.754 | mg/l
0 | Organism\|PeripheralVenousBlood\|Caffeine\|Plasma (Peripheral Venous Blood) | t_max | 0.05 | h
0 | Organism\|PeripheralVenousBlood\|Caffeine\|Plasma (Peripheral Venous Blood) | C_tEnd | 8.155916214 | µmol/l
0 | Organism\|PeripheralVenousBlood\|Caffeine\|Plasma (Peripheral Venous Blood) | AUC | 12058.71387 | µmol*min/l
0 | Organism\|PeripheralVenousBlood\|Caffeine\|Plasma (Peripheral Venous Blood) | AUC_norm | 2.3418E+12 | µg*min/l

Reply:
Why does the exporter need both, the results and the simulation?

Reply:
Very good question. This is what Core requires, probably because of some meta data being retrieved by path (such as dimension etc..).However the info is already in the column. I need to double check

Reply:
I just checked.Dimension is not saved in the column. Just the path to the quantity. THat's why it's required